depmapomics/upload.py

```diff
@@ -0,0 +1,1298 @@
+        (transcripts_expected_count_renamed.sum(1) != 0)
+        & (transcripts_expected_count_renamed.var(1) != 0)
+    ]
+    transcripts_expected_count_renamed.to_csv(
+        folder + "expression_transcripts_expected_count.csv"
+    )
+
+    # upload to taiga
+    print("Expression: uploading to taiga")
+    tc.update_dataset(
+        dataset_id=taiga_virtual,
+        changes_description="adding profile-level expression data",
+        upload_files=[
+            {
+                "path": folder + "expression_genes_expected_count.csv",
+                "name": "CCLE_RNAseq_reads",
+                "format": "NumericMatrixCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": folder + "expression_transcripts_tpm.csv",
+                "name": "CCLE_RNAseq_transcripts",
+                "format": "NumericMatrixCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": folder + "expression_genes_tpm.csv",
+                "name": "CCLE_expression_full",
+                "format": "NumericMatrixCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": folder + "expression_proteincoding_genes_tpm.csv",
+                "name": "CCLE_expression",
+                "format": "NumericMatrixCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": folder + "expression_proteincoding_genes_expected_count.csv",
+                "name": "CCLE_expression_proteincoding_genes_expected_count",
+                "format": "NumericMatrixCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": folder + "expression_transcripts_expected_count.csv",
+                "name": "CCLE_expression_transcripts_expected_count",
+                "format": "NumericMatrixCSV",
+                "encoding": "utf-8",
+            },
+        ],
+        upload_async=False,
+        add_all_existing_files=True,
+    )
+    if include_ssgsea:
+        print("ssGSEA: loading")
+        enrichments = pd.read_csv(folder + "/gene_sets_all.csv", index_col=0)
+        print("ssGSEA: subsetting and renaming")
+        enrichments_renamed = enrichments[enrichments.index.isin(prs)]
+        enrichments_renamed.to_csv("temp/enrichments_ssGSEA.csv")
+        print("ssGSEA: uploading to taiga")
+        tc.update_dataset(
+            dataset_id=taiga_virtual,
+            changes_description="adding enrichments",
+            upload_files=[
+                {
+                    "path": "temp/enrichments_ssGSEA.csv",
+                    "name": "CCLE_ssGSEA",
+                    "format": "NumericMatrixCSV",
+                    "encoding": "utf-8",
+                },
+            ],
+            upload_async=False,
+            add_all_existing_files=True,
+        )
+
+
+def uploadExpressionMatricesModel(
```

Comment:
I don't see the upload like this. All of these functions are doing the same thing and are copy/paste of the same code. I think it should just be one function per action. 
1. you give it a file to subset & rename (per file)
2. you give it a list of files you want to upload (for the entire list of files)